src/components/Verse/OverflowVerseActionsMenuBody.tsx

```diff
@@ -251,7 +260,11 @@ const OverflowVerseActionsMenuBody: React.FC<Props> = ({
         onActionTriggered={onActionTriggered}
       />
 
-      <PopoverMenu.Item onClick={onToggleBookmarkClicked} icon={bookmarkIcon}>
+      <PopoverMenu.Item
+        onClick={onToggleBookmarkClicked}
+        icon={bookmarkIcon}
+        isDisabled={isVerseBookmarkedLoading}
```

Comment:
We are disabling the button when the data is still loading. So it will be less likely to have a bookmark out of sync error